volim.css

```diff
@@ -333,4 +333,76 @@ body {
   font-size: 20px;
   margin-top: 10px;
   color: #252121;
-}
\ No newline at end of file
+}
+
+.shortcode{
+  display: block;
+  height: 700px;
+  width: 100%;
+}
+
+.shortcode-top{
+  display: block;
+  height: 17%;
+  width: 100%;
+}
+
+.shortcode-title{
+  display: block;
+  width: 300px;
+  height: 60px;
+  border-bottom: 1px solid black;
+  position: relative;
+  margin: auto;
+}
+
+.shortcode-title__text{
+  font-size: 45px;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 600;
+  text-align: center;
+}
+
+.shortcode-title__circle{
+ display: block;
+ width: 15px;
+ height: 15px;
+ position: absolute;
+ bottom: -8px;
+ right: 140px;
+ background: #b52f3a;
+ border-radius: 50%;
+}
+
+.shortcode-bottom{
+  display: block;
+  height: 77%;
+  width: 100%;
+}
+
+.shortcode-left{
+  display: inline-block;
+  width: 40%;
+  height: 100%;
+}
+
+.shortcode-left__img{
+  width: 80%;
+  height: 100%;
+  margin-left: 150px;
+}
+
+.shortcode-right{
+  display: inline-block;
+  width: 58%;
+  height: 100%;
+  float: right;
+  position: relative;
+}
+.shortcode-right__img{
+  display: block;
+  width: 65%;
+  height: 90%;
+  position: absolute;
+  right: 250px;
```

Comment:
do not use code like that, better to use %, and don't know why you used `absolute`?